ci-operator/step-registry/servicemesh/kiali/cypress-tests/execute/servicemesh-kiali-cypress-tests-execute-commands.sh

```diff
@@ -36,6 +36,8 @@ export CYPRESS_AUTH_PROVIDER="kube:admin"
 export CYPRESS_RETRIES=2
 export TEST_GROUP="not @crd-validation and not @multi-cluster and not @skip-lpinterop"
 yarn cypress:run:test-group:junit || true # do not fail on a exit code != 0 as it matches number of failed tests
+# save screenshots from the 1st run
+cp -r cypress/screenshots ${ARTIFACT_DIR}/ || true
```

Comment:
Are you sure it will not be overwritten later or the files are unique?

Reply:
yes, the files are unique (the name of the folder is the name of the test). First, all tests without `@crd-validation` are running and then only `@crd-validation` (istio_config.feature) are running.

Right now, it looks like the second yarn cypress:run... clear all folders from the previous run, so only istio_config.feature screenshots are archived, e.g. see here (where lots of cypress tests failed) 
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift_release/55677/rehearse-55677-periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.16-lp-interop-servicemesh-interop-ibmcloud/1829188400041693184/artifacts/servicemesh-interop-ibmcloud/servicemesh-kiali-cypress-tests-execute/artifacts/screenshots/istio_config.feature/